### PR TITLE
[DOC DAY] Chat Services

### PR DESCRIPTION
**Pull request by klshuster**
**Patch description**
Add some more explicit instructions about how to use the `config.yml` and also how the `models` key works.

### CODE REVIEW

File: docs/source/tutorial_chat_service.md [hunk 1]
```diff
@@ -11,13 +11,15 @@ We currently support the following chat services:
...
+:::{note}
+If you'd like to use a service outside of the 4 listed above, please read [here](https://github.com/facebookresearch/ParlAI/tree/master/parlai/chat_service) for information on how to set up a new chat service.
+:::
 
 ## Overview
 
-As stated, humans messaging on chat services can be viewed as a type of agent in ParlAI, communicating with models via observations and action dicts. See [here](https://github.com/facebookresearch/ParlAI/blob/master/parlai/chat_service/core/agents.py) for the basic implementation. Human agents are placed in worlds with ParlAI agent(s) and possibly other humans, and the world defines how each of these agents interacts.
+As stated, humans messaging on chat services can be viewed as a type of agent in ParlAI, communicating with models via observations and action dicts. You can find the implementation of `ChatServiceAgent`s [here](https://github.com/facebookresearch/ParlAI/blob/master/parlai/chat_service/core/agents.py). Human agents are placed in worlds with ParlAI agent(s) and possibly other humans, and the world defines how each of these agents interacts.
```
Comment (stephenroller) [inline docs/source/tutorial_chat_service.md:20]: do you want chat_service to be part of the API reference? See what's done in api.md and the core folder if you want.

File: docs/source/tutorial_chat_service.md [hunk 2]
```diff
@@ -44,6 +46,54 @@ additional_args:
...
+- **`task_name`** - the name of your task. not super important to define, but good to have nonetheless
+- **`world_module`** - this is a module path to the module in which **all** of the task worlds are defined. If you have worlds in separate files, simply importing them into this file will suffice.
+- **`overworld`** - The name of the overworld class.
+- **`max_workers`** - How many max human connections you plan on supporting for your service
+- **`opt`** - any task-specific options you need to define.
+    - **`models`** - Perhaps **the most important** key in this config. This is where you define the models that you'll be using. Each entry under this is a mapping from custom model name to model params. So, in this example, we have defined a model `blender_90M` that we would like to access in their task. For this model, we then specify the actual model arguments necessary to create a ParlAI agent.
+- **`additional_args`** - These are for any specific service-specific args you might want to specify - e.g. if using Facebook Messenger, you would specify your page ID here.
+
+:::{important}
+**ON THE SUBJECT OF MODELS**
```
Comment (stephenroller) [inline docs/source/tutorial_chat_service.md:65]: If you put this on the previous line, it will be the "header" of the note

**Review by stephenroller — APPROVED** [review]

File: docs/source/tutorial_chat_service.md (hunk 1 above)
Comment (klshuster) [reply docs/source/tutorial_chat_service.md:20]: i can't find any `api.md` file?

File: docs/source/tutorial_chat_service.md (hunk 2 above)
Comment (klshuster) [reply docs/source/tutorial_chat_service.md:65]: Just tried that and I think I prefer this way

File: docs/source/tutorial_chat_service.md (hunk 1 above)
Comment (stephenroller) [reply docs/source/tutorial_chat_service.md:20]: ah sorry it's core.md

**Comment by klshuster** [comment]
website build is failing because the websocket chat service requires a `tornado` installation, are we comfortable adding that to `requirements.txt`?

**Comment by stephenroller** [comment]
Yeah someone else complained that tornado wasn’t in requirements so let’s add it

**Comment by klshuster** [comment]
added tornado, will merge when tests pass

**Pull request merged**